From ae401b66d6439bf32bbc1e446bceb44e8fffc014 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 14 Apr 2022 13:18:43 +0300
Subject: [PATCH] [SW-0] gaudi3: fix notify slave check

This patch fixes condition to call notify slave function in case
of dual die sku (sku = 0) and master die (die_id = 0).

Change-Id: I08337ee78e195e0ab87a212bde76a6834aee8761
---

diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index 3e7daf9..0fe2d25 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -188,7 +188,7 @@
 	arcboot_rom_status_update(PROGRESS_ARCBOOT_IN_RECOVERY);
 
 	/* Notify slave if master die and dual die sku */
-	if ((arcboot_rom_global.sku) && (!arcboot_rom_global.die_id))
+	if ((!arcboot_rom_global.sku) && (!arcboot_rom_global.die_id))
 		arcboot_rom_notify_slave();
 
 	/*
